### ISSUE TITLE
No connection after upgrading to 2.5

### ISSUE THREAD

**Issue by heyarne** [nextcloud/desktop#868]
### Expected behaviour
I have two accounts set up that I'm using at the same time; one is for an older Nextcloud instance that I don't control and don't know the version of, one is for one which I do control and keep up to date. I expected to be able to use both after upgrading just like before.

### Actual behaviour
I can't connect to the latter. While it just showed me "No connection" for a while. When I log out explicitly and re-open the client, an authentication popup is shown. After authenticating it's stuck giving me a message saying "Connecting as *heyarne*...". My avatar shows up but it keeps on connecting indefinitely.

### Client configuration
Client version:

```
$ nextcloud --version
Nextcloud version 2.5.0git
Git revision 371001c9ffea5e99abe971fcb155af64f25b024b
Using Qt 5.11.2, built against Qt 5.11.2
Using 'OpenSSL 1.1.1  11 Sep 2018'
```

Operating system: Arch Linux

OS language: English

Qt version used by client package (Linux only, see also Settings dialog): see above

Client package (From Nextcloud or distro) (Linux only):
```
$ yay -Qi nextcloud-client
# https://www.archlinux.org/packages/community/x86_64/nextcloud-client/
Name            : nextcloud-client
Version         : 2.5.0-4
Description     : Nextcloud desktop client
Architecture    : x86_64
URL             : https://nextcloud.com/
Licenses        : GPL
Groups          : None
Provides        : None
Depends On      : sqlite  qtkeychain  qt5-svg  qt5-webengine  qt5-webkit  xdg-utils
Optional Deps   : kio: integration with Dolphin
                  nemo-python: integration with Nemo
                  python-nautilus: integration with Nautilus
                  python2-caja: integration with Caja
Required By     : None
Optional For    : None
Conflicts With  : None
Replaces        : None
Installed Size  : 9.65 MiB
Packager        : Chih-Hsuan Yen <yan12125@archlinux.org>
Build Date      : Sat 17 Nov 2018 03:49:31 PM CET
Install Date    : Tue 20 Nov 2018 08:31:50 PM CET
Install Reason  : Explicitly installed
Install Script  : No
Validated By    : Signature
``` 

Installation path of client: `/usr/bin/nextcloud`

### Server configuration

Operating system: Ubuntu 18.04.1 LTS (GNU/Linux 4.15.0-39-generic x86_64)

Web server: nginx/1.14.0 (Ubuntu)

Database: mysql  Ver 15.1 Distrib 10.1.34-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2

PHP version: PHP 7.0.32-4+ubuntu18.04.1+deb.sury.org+1

Nextcloud version: 14.0.3

Storage backend (external storage):

### Logs

1. Client log
I'm not sure what I can post here. `grep $server_url nextcloud.txt`?

2. Web server error log:
```
Error | end_to_end_encryption | Can't create public key: could not sign the CSR, please make sure to submit a valid CSR | 2018-11-20T21:04:42+0100
-- | -- | -- | --
Error | PHP | openssl_csr_sign():  cannot get CSR from parameter 1 at  /var/www/nuee.schlueter.is/apps/end_to_end_encryption/lib/SignatureHandler.php#60 | 2018-11-20T21:04:42+0100
Debug | no app in context | No  cache entry found for  /appdata_ocjx58mjh67j/end_to_end_encryption/public-keys/arne.public.key  (storage: local::/mnt/nextcloud/, internalPath:  appdata_ocjx58mjh67j/end_to_end_encryption/public-keys/arne.public.key) | 2018-11-20T21:04:42+0100
Debug | no app in context | No  cache entry found for  /appdata_ocjx58mjh67j/end_to_end_encryption/public-keys/arne.public.key  (storage: local::/mnt/nextcloud/, internalPath:  appdata_ocjx58mjh67j/end_to_end_encryption/public-keys/arne.public.key) | 2018-11-20T21:04:42+0100
```

**Comment by bonanza123** [comment]
I guess I had a similar problem, did you try disabling the end to end encryption app in the server?

**Comment by heyarne** [comment]
Indeed, that fixed it. It seems like I have to keep the E2E-App disabled, is that correct? Is this mentioned somewhere and is someone working on a fix?

**Comment by MegaCookie** [comment]
Disabling end to end encryption on the server was the trick. However it's a pity that we cannot try end to end encryption with this client yet.

**Comment by digver** [comment]
Thanks, I had a similar problem, two accounts on the same server, one account one not connect. Disabling the E2E app solved it.

**Comment by eknoes** [comment]
Solved it for me too.

**Comment by bonanza123** [comment]
Problem persists with NC15 and client version 2.5.1.

**Comment by DominiqueFuchs** [comment]
Duplicate of original issue [desktop/#830](https://github.com/nextcloud/desktop/issues/830) and fixed by [desktop/PR#1420](https://github.com/nextcloud/desktop/pull/1420)
